### ISSUE TITLE
Panning on the map has no effect

### ISSUE THREAD

**Issue by bilogic** [ondras/wwwsqldesigner#310]
There is a little map on the lower right corner of the editor, but the main window contents doesn't update when moving the small box. Is this by design? Thank you.

**Comment by ondras** [comment]
I cannot reproduce; the minimap seems to be correctly reflected in the main window. What URL are you trying, what browser, is anything relevant displayed in the Console?

**Comment by bilogic** [comment]
Thanks for getting back!

I might have been a little unclear in my OP. Let me try again:

1. Chrome Version 100.0.4896.60 (Official Build) (64-bit)
2. If I use scrollbars to move my main window around, the minimap reflects correctly
3. If I drag the little box around in the minimap, the main window doesn't move and match up to the minimap
4. There is 1 error in the console, but I doubt it is the issue: `Failed to load resource for favicon.ico`

Makes sense?

**Comment by ondras** [comment]
Yes, I can confirm that problem (in Chrome) now. It works okay in Firefox. Will have a look later.

**Comment by bilogic** [comment]
if it helps, https://github.com/Lt-Mayonesa/wwwsqldesigner works in Chrome

**Comment by ondras** [comment]
Should be fixed now.

**Comment by bilogic** [comment]
Thanks, https://ondras.zarovi.cz/sql/demo/?keyword=default works!

Should I file a PR to auto format all whitespaces in a standardized way? This will eliminate all the unnecessary whitespaces in future PRs. See #314
